docs/content/en/docs/packages/packagelist.md

```diff
@@ -5,17 +5,23 @@ weight: 6
 description: >
   List of EKS Anywhere curated packages
 ---
+### What's New
+
+#### MetalLB (v0.14.8)
```

Comment:
Should we do it based on bundle of packages somehow? This tag for MetalLB is based on the upstream version, not necessarily our version right?